### ISSUE TITLE
Translation: avoid mentioning future features

### ISSUE THREAD

**Issue by deejam** [HaikuArchives/BePDF#88]
Change "Editing of annotations in an encrypted PDF file isn't supported yet!" -> "Editing of annotations in an encrypted PDF file isn't supported!" (the word "yet" was removed).

**Comment by humdingerb** [comment]
I'm not sure this minor change is worth invalidating the string for all languages. If done, I suggest to not to wait on the translations of it for the next release.

**Comment by deejam** [comment]
> I'm not sure this minor change is worth invalidating the string for all languages. If done, I suggest to not to wait on the translations of it for the next release.

All errors should be fixed. If there are problems with the release routines, they should be fixed.

**Comment by humdingerb** [comment]
I wouldn't categorize this change as error. It's a probably too optimistic view of the future... :)
What problem do you see in the release routine? Waiting with a release until everyone has re-translated that string sans "yet" can delay the release by months.

**Comment by deejam** [comment]
> I wouldn't categorize this change as error. 

You're right. The reply was not for this ticket, but more general for my other tickets created today. So I should have written it in another ticket. My bad.

> What problem do you see in the release routine?

I don't know how the release routine works.

>  Waiting with a release until everyone has re-translated that string sans "yet" can delay the release by months.

Usually you do it the other way around; a string freeze. Since the translations is in a single repository, changes has to be made after the freeze by creating issues. Any changes can be committed after the freeze.

It would be good if translations could be freezed for each release. In that way, translations could be made in master ("nightly") without affecting upcoming releases.
